qa_bands.md

```diff
@@ -106,9 +110,8 @@
 ## Principal QA Engineer
 ### Technical Abilities
 * Mastery of TuneIn's automation frameworks
-* Mastery of TuneIn's application and systems architecture
-* Is an authority in multiple test technical disciplines, languages, frameworks, and systems
-* Demonstrates a unique and extraordinarily effective problem solving perspective and approach
```

Comment:
True, but wouldn't that be something that falls under project scope as opposed to technical abilities?